src/store/actionsV2/searchTrialInvestigatorsAction.js

```diff
@@ -0,0 +1,30 @@
+import { ACTIVE_TRIAL_STATUSES } from '../../constants';
+
+/**
+ * Gets a list of Trial Investigators from the CTS api
+ * @param {String} searchText - free text string matching trial investigator name
+ * @return {{payload: {cacheKey: string, method: string, service: string, requestParams: {agg_field: string, agg_name: string, current_trial_status: string[], include: string[]}}, type: string}}
+ */
+export const searchTrialInvestigatorsAction = ({ searchText }) => {
+	if (!searchText || searchText === '') {
+		throw new Error(
+			'You must specify a trial investigator in order to fetch it.'
+		);
+	}
+	const requestQuery = {
+		agg_field: 'principal_investigator',
+		agg_name: searchText,
+		current_trial_status: ACTIVE_TRIAL_STATUSES,
+		include: ['none'],
```

Comment:
do we need to include: none? Just wondering what results will be returned without that param, or what is the difference between not providing include:none and providing it ?

Reply:
I only included it since the v2 request params had it included, I removed it and tested it and it seems to be fine. In leadOrgAction.js there is also a include: ['none'] param, should I remove that too?